querybook/server/logic/user.py

```diff
@@ -194,6 +194,8 @@ def create_or_update_user_group(user_group: UserGroup, commit=True, session=None
             [UserGroupMember(gid=group.id, uid=user_id) for user_id in members_to_add]
         )
 
+    update_es_users_by_id(group.id)
```

Comment:
iirc user groups are not added to es right now, not sure if we should also update the logic here as well.
https://github.com/pinterest/querybook/blob/abd32a141d6235ec677561af48a9b556db764f96/querybook/server/logic/elasticsearch.py#L724

Reply:
The logic there is already updated to remove the `if not user.is_group` conditional. Since user groups are essentially the same as users, I think they should be updated in es with the conditional removed